netsim/devices/cumulus.py

```diff
@@ -15,10 +15,24 @@ def check_ospf_vrf_default(node: Box) -> None:
         category=log.IncorrectType,
         module='quirks')
 
+"""
+Checks whether any node VLAN is in the (changeable) reserved range
+"""
+def check_reserved_vlan_range(node: Box) -> None:
+  for vname,vdata in node.get('vlans',{}).items():
+    vid = vdata.get('id',0)
+    if vid>=3725 and vid <=3999:
```

Comment:
Where would Cumulus Linux 4.x enforce that limitation? We're configuring VLANs "by hand"

Reply:
n/a